lib/media/track/remotevideotrack.js

```diff
@@ -37,6 +37,18 @@ class RemoteVideoTrack extends RemoteMediaVideoTrack {
   toString() {
     return `[RemoteVideoTrack #${this._instanceId}: ${this.sid}]`;
   }
+
+  /**
+   * Update the subscribe {@link Track.Priority} of the {@link RemoteVideoTrack}.
```

Comment:
Would `Update the subscribed {@link Track.Priority}` be better?

Reply:
@timmydoza The current wording is the correct one, since we are talking about the subscribe priority of the Track. If we were talking about the Track instead of the priority, we would say "subscribed Track's priority".

Reply:
Sounds good :+1:.  Thanks!